### ISSUE TITLE
Support for -p parameter

### ISSUE THREAD

**Issue by hanny24** [avast/docker-compose-gradle-plugin#56]
It would be really beneficial for us (ff :)) to have a support for -p parameter. It would solve us a problem when there are multiple jobs running concurrently and trying to create containers with same name.

**Comment by augi** [comment]
No problem with implementing this feature. I'm just curios - what is the environment where do you want to execute more `docker-compose` command for the same directory name?

**Comment by mixalturek** [comment]
Another possibility is environment variable COMPOSE_PROJECT_NAME according to https://github.com/docker/compose/issues/2982.

**Comment by augi** [comment]
The `docker-compose` is executing using [exec](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Exec.html) that by-default uses the same environment as the executing process. So the `COMPOSE_PROJECT_NAME` should work for you IMHO.

**Comment by mixalturek** [comment]
There are parallel builds on our integration server, for example "build" and "release" jobs are both triggered by a commit to the master branch. The builds influence each other as they share the containers now, one of them nearly always fails.

**Comment by augi** [comment]
OK, understand. We've never run into this issue because we have Compose File in top-level directory and TeamCity working directory is a hash.

**Comment by dmitrygusev** [comment]
We solve this with `COMPOSE_PROJECT_NAME` like this:

```groovy
def composeProjectName = ... // modify based on current branch and other conditions

dockerCompose {
    useComposeFiles = ...
    environment.put 'COMPOSE_PROJECT_NAME', composeProjectName
}
```

**Comment by augi** [comment]
Released as 0.3.18.
